native/src/main/resources/META-INF/native-image/io.ballerina.stdlib/os-native/reflect-config.json

```diff
@@ -0,0 +1,6 @@
+[
+{
+  "name":"java.util.Collections$UnmodifiableMap",
```

Comment:
Can we format like this :
```json
[
   {
      "name": "java.util.Collections$UnmodifiableMap",
      "fields": [
         {
            "name": "m"
         }
      ]
   }
]
```

Reply:
Updated in [3665997](https://github.com/ballerina-platform/module-ballerina-os/pull/457/commits/3665997fc630f2f4bc10333c04389359386b0fd1)